core/che-core-metrics-core/pom.xml

```diff
@@ -21,6 +21,15 @@
     </parent>
     <artifactId>che-core-metrics-core</artifactId>
     <name>Che Core :: Metrics :: Core</name>
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.github.mweirauch</groupId>
+                <artifactId>micrometer-jvm-extras</artifactId>
+                <version>0.1.3</version>
```

Comment:
Could you please add a comment here why this dependency should be declared in a dedicated `che-core-metrics-core` dependency management section but not in che parent along with other dependencies?

Reply:
I'll move this dependency to che-parent before the merge